assets/js/modules/adsense/components/dashboard/AdSenseDashboardWidget.js

```diff
@@ -208,8 +208,14 @@ class AdSenseDashboardWidget extends Component {
 							) }>
 								<Layout
 									header
-									/* translators: %s: date range */
-									title={ sprintf( __( 'Performance over the last %s', 'google-site-kit' ), currentDateRange ) }
+									title={ sprintf(
+										/* translators: %s: date range */
+										_n(
+											'Performance over the last %s day',
+											'Performance over the last %s days',
+											currentDateRange, 'google-site-kit'
+										), currentDateRange
```

Comment:
Let's keep usage of the WP translation functions in a single line, even if they become a bit long, similar to how we do it elsewhere in JS and PHP. What we could move to a separate line though is the `currentDateRange` variable passed to `sprintf`.